tinymce/urls.py

```diff
@@ -1,17 +1,18 @@
 # Copyright (c) 2008 Joost Cassee
 # Licensed under the terms of the MIT License (see LICENSE.txt)
+from tinymce.views import textareas_js, spell_check, flatpages_link_list, compressor, filebrowser, preview
```

Comment:
Can we do: ``from tinymce import views`` and then use ``views.textareas_js`` in the url definition?